modules/adxpremiumAnalyticsAdapter.js

```diff
@@ -125,16 +210,36 @@ function deviceType() {
   return 'desktop';
 }
 
+function clearSlot(elementId) {
+  if (elementIds.includes(elementId)) { elementIds.splice(elementIds.indexOf(elementId), 1); utils.logInfo('AdxPremium Analytics - Done with: ' + elementId); }
+  if (elementIds.length == 0 && !requestSent && !timeoutBased) {
+    requestSent = true;
+    sendEvent(completeObject);
+    utils.logInfo('AdxPremium Analytics - Everything ready');
+  }
+}
+
+function sendEventFallback() {
+  setTimeout(function () {
+    if (!requestSent) { requestSent = true; sendEvent(completeObject); utils.logInfo('AdxPremium Analytics - Sending event using fallback method.'); }
+  }, 2000);
+}
+
 function sendEvent(completeObject) {
+  requestDelivered = true;
   try {
     let responseEvents = btoa(JSON.stringify(completeObject));
     let mutation = `mutation {createEvent(input: {event: {eventData: "${responseEvents}"}}) {event {createTime } } }`;
     let dataToSend = JSON.stringify({ query: mutation });
-    ajax(url, function () { utils.logInfo(Date.now() + ' Sending event to adxpremium server.') }, dataToSend, {
+    let ajaxEndpoint = defaultUrl;
+    if (adxpremiumAnalyticsAdapter.initOptions.sid) {
```

Comment:
is `sid` some adapter parameter? Can you mention it in the `adxpremiumAnalyticsAdapter.md` so we have it documented somewhere?

Reply:
Yes, sid = Assigned Server ID since we have multiple analytics servers. Will be mentioned in the markdown in a moment.

Reply:
Done.